### ISSUE TITLE
Are there any changes on the original Bootstrap-Files?

### ISSUE THREAD

**Issue by ghost** [SU-SWS/open_framework#156]
... Because with Services like paintstrap.com you can create your own colour profile for Bootstrap.
The Output are Standard-Bootstrap-Files.

Questions: When i use Paintstrap and override the Bootstrap-Files. Does this affect OpenFramework? 
Are there any changes on the original Bootstrap-Files?

Because i would like to change the Colours.

**Comment by hyperboy** [comment]
Yes, we made some minor modifications to the bootstrap CSS files, specifically boostrap-responsive.min.css and bootstrap-responsive.css.

In bootstrap-responsive.css, we updated the media query values on lines 952 and 1104, changing the values from @media (max-width: 979px) and @media (min-width: 980px) to @media (max-width: 767px) and @media (min-width: 768px) respectively.  We also made the same change in bootstrap-responsive.min.css.

We received feedback that users preferred viewing the entire main menu navigation in tablet view, so we updated the media query values such that the main menu navigation would transform into a drawer at the 767px break point instead of the 979px break point.

If you plan to leverage http://paintstrap.com, it's okay to add the paintstrap modified bootstrap files in your subtheme, overriding the bootstrap files in Open Framework.  

```
stylesheets[all][] = bootstrap/css/bootstrap.min.css
stylesheets[all][] = bootstrap/css/bootstrap-responsive.min.css
```

You'll probably want to manually edit the media query values in bootstrap-responsive.min.css afterwards.
